packages/bemjson-to-jsx/lib/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var bn = require('@bem/sdk.naming');
+var bn = require('@bem/sdk.naming.entity');
```

Comment:
`bn`, `bemNaming` and `BemNaming` in different packages

Reply:
→ https://github.com/bem/bem-sdk/issues/212